From cf99d5adf177418ca4093be8ff0a30364a739089 Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Mon, 20 Dec 2021 12:49:59 +0800
Subject: [PATCH] Add QMP client which syncs guest RTC settings to host

Add new QMP host utility guest_rtc_monitor
into build.

Tracked-On: OAM-100443
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
---
 groups/device-specific/caas/AndroidBoard.mk | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/groups/device-specific/caas/AndroidBoard.mk b/groups/device-specific/caas/AndroidBoard.mk
index 052fad5..edc29a2 100644
--- a/groups/device-specific/caas/AndroidBoard.mk
+++ b/groups/device-specific/caas/AndroidBoard.mk
@@ -26,5 +26,10 @@ em-host-utilities:
 	cd device/intel/civ/host/backend/thermal/vm_thermal_utility && $(LOCAL_MAKE)
 	cp device/intel/civ/host/backend/thermal/vm_thermal_utility/thermsys $(PRODUCT_OUT)/scripts/
 
+.PHONY: rtc-monitor
+rtc-monitor:
+	cd device/intel/civ/host/rtc-monitor && $(LOCAL_MAKE)
+	cp device/intel/civ/host/rtc-monitor/guest_rtc_monitor $(PRODUCT_OUT)/scripts/
+
 .PHONY: host-pkg
-host-pkg: em-host-utilities vinput-manager
+host-pkg: em-host-utilities vinput-manager rtc-monitor
-- 
2.17.1

